### PR TITLE
Fix EZP-25292: fix additional check

### PR DESCRIPTION
**Pull request by guillaumelecerf**

### CODE REVIEW

**Comment by andrerom** [comment]
+1, ping @joaoinacio

File: update/common/scripts/5.4/cleanuntranslatablerelations.php [hunk 1]
```diff
@@ -53,7 +53,8 @@
  */
 function parseRelationListIds( $relationListXml )
 {
-    if ( empty( trim( $relationListXml ) ) )
+    $trimmedRelationListXml = trim( $relationListXml );
```
Comment (guillaumelecerf) [inline update/common/scripts/5.4/cleanuntranslatablerelations.php]: Maybe should be `$relationListXml = trim( $relationListXml );` ?

**Comment by joaoinacio** [comment]
It's OK, the alternative option would be to leave master as-is (php 5.5, cleaner IMHO) and for 5.4 perform the comparison using `if ( trim( $relationListXml ) == false )` as PHP officially recommends for this case.

**Comment by guillaumelecerf** [comment]
@joaoinacio : what would you prefer ?

**Comment by andrerom** [comment]
from my side: lets try to avoid having differences between master and backports, also note that master for legacy supports php 5.3.3 and up as it won't receive further major releases.

File: update/common/scripts/5.4/cleanuntranslatablerelations.php (hunk 1 above)
Comment (andrerom) [inline update/common/scripts/5.4/cleanuntranslatablerelations.php]: can do that

**Comment by joaoinacio** [comment]
OK given the above it's a no-brainer, +1 as-is

**Comment by guillaumelecerf** [comment]
Thanks.

**Pull request merged**